docs/.vuepress/components/HelloWorld.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div> 
+    <tabs :options="{ useUrlFragment: false }"  @changed="tabClicked">
+      <tab name="JavaScript" id="js">
+        <iframe v-if="selected === 'js'" :src="jsSrc"
+          :style="styleAttribute"
+          :allow="allowAttribute"
+          :sandbox="sandboxAttribute"
+          title="Handsontable JavaScript Data Grid - Hello World App"
+        ></iframe>
```

Comment:
As long as we have only 3 tabs, this comment is a low-important proposal.
Each used iframe has similar construction, it is a perfect situation to extract it as a separated component `CodeSandboxIframe` with a parameter `id`.

Example:
```js
<CodeSanboxIframe id="handsontable-javascript-data-grid-hello-world-app-dzx8f" />
```

However, as I mentioned before. This is not crucial for this project to extract this component.
So, if you want to do it a little better? - just do it. If won't, feel good to ignore this proposal.

Reply:
I will ask you for help with that. I prepared some code, but I have some problems. I suggest to do it within next PR.

Reply:
ok:+1